### DISCUSSION TITLE
when is game info displayed?

### DISCUSSION THREAD

**Discussion by mdskier** [MLB-LED-Scoreboard/mlb-led-scoreboard#450] [Q&A] [answered]
I have successfully installed this code and was not sure if there was something wrong. I was only getting the standings screens (and news if I enabled it). But no game information. Then I noticed game info being displayed and the live game info during a live game.So it seems that game info is only shown when game time is near?
 I thought it would cycle thru game info if there was a game scheduled, which would let me know when the game was starting, etc.

**Comment by WardBrian** [comment, marked as answer]
Inside `config.json` there is a setting underneath `"standings"` called `"display_no_games_live"`. You can set that to `false` if you'd like to see the games all day long

**Reply by mdskier** [reply]
That helps tremendously. Thanks!
